daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -59,7 +59,7 @@ message PackageRef {
 
     // An index into `interned_package_ids` of the Package containing
     // this reference.
-    uint64 interned_id = 3; /* Available since version 1.dev */
```

Comment:
We need to adjust the minor version history above as well.

Reply:
Good catch. Fixed.